### PR TITLE
Convert CSS to per-component model

### LINKED ISSUE #420: refactor custom css
break out custom css per component/container to live along side the objects it styles

### PR DESCRIPTION
**Pull request by cailyncodes**
Closes #420

### CODE REVIEW

File: src/Hedwig/ClientApp/package.json [hunk 1]
```diff
@@ -58,6 +59,7 @@
 		"@types/storybook__addon-storyshots": "^5.1.1",
 		"@types/storybook__react": "^4.0.2",
 		"babel-plugin-macros": "^2.6.1",
+		"classnames": "^2.2.6",
```
Comment (cmgiven) [inline src/Hedwig/ClientApp/package.json]: Ahh yes, it's time.

File: src/Hedwig/ClientApp/src/assets/styles/index.scss [hunk 2]
```diff
@@ -35,4 +34,10 @@
 // Import theme custom styles
 
 @import 'uswds-theme-custom-styles';
-@import 'print';
+
+@import 'general';
+@import '../../components/Header';
+@import '../../components/InlineIcon';
+@import '../../components/StepList';
+@import '../../components/Table';
```
Comment (cmgiven) [inline src/Hedwig/ClientApp/src/assets/styles/index.scss]: Suggestion: Maybe an index.scss in /components that can be imported as a single item here?

File: src/Hedwig/ClientApp/src/components/Header/Header.tsx [hunk 3]
```diff
@@ -29,6 +30,7 @@ const setActiveStateOfNavItem = function(item: NavItemProps, index: number, path
 
 	return { ...item, active };
 };
+// console.log(styles);
```
Comment (cmgiven) [inline src/Hedwig/ClientApp/src/components/Header/Header.tsx]: Reminder to tidy.

File: src/Hedwig/ClientApp/src/components/Header/Header.tsx [hunk 4]
```diff
@@ -72,46 +74,54 @@ class HeaderWithoutRouter extends React.Component<HeaderProps & RouteComponentPr
...
+						'usa-overlay',
+						{ 'is-visible': this.state.menuIsVisible }
+					)}
 					onClick={this.hideMenu}
 				></div>
-				<header className="usa-header usa-header--extended" role="banner">
-					<div className="usa-navbar">
-						<div className="usa-logo" id="extended-logo">
+				<header className={cx("usa-header", "usa-header--extended")} role="banner">
+					<div className={cx("usa-navbar")}>
```
Comment (cmgiven) [inline src/Hedwig/ClientApp/src/components/Header/Header.tsx]: Is it necessary to do this when there's just a single class?

**Comment by cailyncodes** [comment]
Awaiting USWDS v2.4.1 (see https://github.com/uswds/uswds/issues/3285; awaiting https://github.com/uswds/uswds/issues/3290 to land) or convert back to node-sass

File: src/Hedwig/ClientApp/src/components/Header/Header.tsx (hunk 4 above)
Comment (cailyncodes) [reply src/Hedwig/ClientApp/src/components/Header/Header.tsx]: Not strictly -- I guess it is excessive to do it prematurely -- I just found and replaced all the strings starting with `className=` within the files I changed

File: src/Hedwig/ClientApp/package.json (hunk 1 above)
Comment (cailyncodes) [reply src/Hedwig/ClientApp/package.json]: Indeed haha

**Comment by cailyncodes** [comment]
@cmgiven This is the rest of the conversion -- another quick look would be appreciated!

File: src/Hedwig/ClientApp/src/containers/Enrollment/Edit/EnrollmentEdit.tsx [hunk 5]
```diff
@@ -107,7 +107,7 @@ export default function EnrollmentEdit({
 		>
 			<div className="grid-container">
 				<h1>Edit {section.name.toLowerCase()}</h1>
-				<p className="usa-intro">{nameFormatter(enrollment.child)}</p>
+				<p className="intro">{nameFormatter(enrollment.child)}</p>
```
Comment (mmazanec22) [inline src/Hedwig/ClientApp/src/containers/Enrollment/Edit/EnrollmentEdit.tsx]: usa-intro is a [USWDS style](https://designsystem.digital.gov/components/typography/#prose) (it's not explained here but it's used on the part about the Great Smoky Mountains)-- are we deliberately getting rid of it?  If so should we call ours oec-intro to distinguish them?

**Pull request merged**